meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx6/lmp-secure.cfg

```diff
@@ -0,0 +1,4 @@
+CONFIG_CMD_FIOVB=y
```

Comment:
Same for this file.

This is just because currently lmp.cfg for apalis-imx6 is mostly good, but that won't necessarily always be the case, because we will instead restrict the options at the sec flavors.

@mike-scott what do you think?